tox.ini

```diff
@@ -41,6 +41,13 @@ deps =
     pytest
     pytest-cov
 
+[testenv:py3-latest]
+deps =
+    pytest
+    pytest-xdist
+commands =
+    {posargs:pytest -n auto -vv tests}
```

Comment:
If we're not sure the tests will pass in <6 hours, we could `--skip-slow` here?

A lot of the slow tests are only slow in lieu of running (essentially a) full simulation. Any breakages in running simulations - by this logic - I would presume would be flagged up by the other tests that run shorter simulations. Or at least, I'd be prepared to sacrifice the slow tests for complete information about whether the non-slow ones passed :sweat_smile:

Plus we'll always be running the full test suite in any PR against `master` that does bump the dependencies, so if something does slip through the cracks we'll still catch it before releasing it into `master`.

Reply:
Yeah agree this is probably worthwhile doing - will update.